src/server/pkg/worker/api_server.go

```diff
@@ -605,13 +619,19 @@ func (a *APIServer) Process(ctx context.Context, req *ProcessRequest) (resp *Pro
 		}
 		return nil, err
 	}
+	logger.Logf("finished uploading output")
 	return &ProcessResponse{
 		Tag: &pfs.Tag{tag},
 	}, nil
 }
 
 // Status returns the status of the current worker.
-func (a *APIServer) Status(ctx context.Context, _ *types.Empty) (*pps.WorkerStatus, error) {
+func (a *APIServer) Status(ctx context.Context, _ *types.Empty) (status *pps.WorkerStatus, retErr error) {
+	logger := log.Logger{}
+	logger.Printf("service: worker, method: Status, response: %v, err %v, duration: %v", status, nil, nil)
+	defer func(start time.Time) {
+		logger.Printf("service: worker, method: Status, response: %v, err %v, duration: %v", status, retErr, time.Since(start))
+	}(time.Now())
```

Comment:
Yea ... but that kind of makes sense. The `getTaggedLogger()` takes a `processRequest` object ... and those logs get metadata associated with running that exact job/datum.

The `Status()` call contains the datum in the response ... so tying it w the metadata for `get-logs` is redundant I think.

I guess the question is would a user want to see these logs (via `get-logs`). I think they might want to see that it's `Cancelled`. I'm not sure they need to see any `Status()` calls ... I think that means you might see more log lines anytime the status is requested.

I opted for the empty logger because it seemed like the simplest change. If we want `Cancel()` to be discoverable by `get-logs` I'll need to decorate the server struct w the datum info from `processRequest`. I think in that case I'll want to put a `sync` around setting/getting those fields as well (since the worker could get a new datum to process at anytime).